test/karma.conf.js

```diff
@@ -30,12 +30,17 @@ const MAX_CIRCLE_CI_CONCURRENCY = 83;
 module.exports = function setKarmaConfig(config) {
   const baseConfig = {
     basePath: '../',
-    browsers: ['ChromeHeadlessNoSandbox'],
+    browsers: ['chromeHeadless'],
```

Comment:
?

Reply:
It's an arbitrary name. I have replicated the name we give in the main repo.

Reply:
The names are relevant to the CLI. You can do `yarn test:karma --browsers browserNameA,browserNameB`.

So the longer the name, the more you have to type in your terminal if you just want to test a subset of browsers. Think of the diff as 
```diff
- yarn test:karma --browsers ChromeHeadlessNoSandbox
+ yarn test:karma --browsers chromeHeadless
```

And since we don't have a `chromeHeadlessSandbox` I don't see why we need the `NoSandbox` qualifier in the first place. I haven't found the information whether we run with or without a sandbox useful.